ImperatorToCK3/Data_Files/configurables/culture_map.txt

```diff
@@ -299,12 +310,15 @@ link = { ck3 = finnish @ir_all_germanic_cultures ck3Province = 454 ck3Province =
 # Karelian, Vepsian skipped because out of scope
 
 # East Germanic mappings
+# Tag culture first so they don't get overwritten
+link = { ck3 = burgundian @ir_vandilian_cultures historicalTag=BUR } # TFE
+link = { ck3 = scirian @ir_vandilian_cultures historicalTag=SCI } # TFE 
+link = { ck3 = varinian @ir_vandilian_cultures @ir_irminonic_cultures historicalTag=VRI } # TFE - Central Germanic in Imperator yet East Germanic in CK3, so use both
+link = { ck3 = rugian @ir_vandilian_cultures historicalTag=RGG } # TFE
+link = { ck3 = herulian @ir_vandilian_cultures historicalTag=X28 } # TFE
```

Comment:
Isn't X28 a dynamic tag? I couldn't find any reference to it in Invictus.

Reply:
It's the tag to the Invictus exclusive tribe of Haria.
![steamuserimages-a akamaihd](https://github.com/ParadoxGameConverters/ImperatorToCK3/assets/87206266/24a88625-3fe0-48e8-8703-0085edea7bb7)

I checked vanilla and there does exist a real Herulia tribe under the tag of HRE but I missed it the first time I checked becuase it doesn't exist in Invictus as it's been split between Ambronia which shares the same tag (HRE) and Eudosia (EUD). Should it just be HRE anyway?

Reply:
Is Haria related to the Herulians?

Reply:
Don't know honestly. I just changed it to HRE.

Reply:
If Eudosia was split off vanilla Herulia, maybe it could be also included, but from my quick research I didn't find a relation. I guess we'd need to ask the Invictus devs who reworked the area. Using the HRE tag seems fine to me, merging!